README.md

```diff
@@ -39,6 +39,11 @@ Josué David Benavides Salinas
 - Gmail:jdbs9514@gmail.com
 - GitHub: @jdbs9514
 
+
+## Acknowledgments
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] Your README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).